go/obscuronode/host/host.go

```diff
@@ -73,16 +77,23 @@ type Node struct {
 
 	// Node nodeDB - stores the node public available data
 	nodeDB *DB
+
+	// A node is marked as ready after it has bootstrapped existing blocks and has the enclave secret
+	readyForWork *int32
+
+	// Handles tx conversion from eth to L1Data
+	txHandler txhandler.TxHandler
```

Comment:
wondering if it makes sense for the Ethereum adapter interface to expose this tx interface as well

Reply:
Can you expand on this please ?